src/klipse/klipse_editors.cljs

```diff
@@ -65,7 +73,13 @@
               (let [result (<! evaluation-chan)
                     results (str previous-results result)]
                 (when (some? result) ;exit if the channel is closed
-                  (setter results)
+                  (if (string? result)
+                    (setter results)
+                    (when-let [klipse-dom-node (js/document.querySelector ".klipse-container")]
```

Comment:
In case `result` is not a string you should call `(setter (second results))`

Reply:
do you think I need to do both dispatch custom event and call (setter (second results))  here ?

Reply:
@dviramontes yes I recommend I do both dispatch custom event and call `(setter (second results))` here